client/src/includes/a11y-result.test.ts

```diff
@@ -55,3 +62,126 @@ describe('sortAxeViolations', () => {
     ]);
   });
 });
+
+describe('customChecks', () => {
+  it('should have function values for each custom check', () => {
+    Object.values(customChecks).forEach((value) => {
+      expect(typeof value).toBe('function');
+    });
+  });
+});
+
+describe('addCustomChecks', () => {
+  it('should integrate custom checks into the Axe spec', () => {
+    const spec: Spec = {
+      checks: [{ id: 'check-id', evaluate: 'functionName' }],
```

Comment:
I don’t understand why `evaluate` is defined here. Shouldn’t this only be added by `addCustomChecks`?